modules/accredited_representative_portal/db/migrate/20241212000100_create_ar_power_of_attorney_requests.rb

```diff
@@ -0,0 +1,9 @@
+class CreateArPowerOfAttorneyRequests < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ar_power_of_attorney_requests, id: :uuid do |t|
+      t.references :claimant, type: :uuid, foreign_key: { to_table: :user_accounts }
+      t.references :latest_resolution, type: :uuid, foreign_key: { to_table: :ar_power_of_attorney_request_resolutions }
```

Comment:
Let's make POA request have one resolution in the app and leave out this denormalization for now. This was that discussion where I said a future migration won't be too painful probably.